test/REQUIRE

```diff
@@ -0,0 +1 @@
+OnlineStats
```

Comment:
Now that OnlineStatsBase depends on LearnBase, I was able to move the entire Series interface and most OnlineStat definitions into OnlineStatsBase.  You can safely remove this file since what you're testing (Mean and LinReg) both live in OnlineStatsBase.

https://github.com/joshday/OnlineStatsBase.jl/blob/master/src/stats.jl

Reply:
Nice!